contracts/cosmwasm-vm/cw-xcall/src/admin.rs

```diff
@@ -41,113 +41,18 @@ impl<'a> CwCallService<'a> {
     /// the `ContractError::AdminAddressCannotBeNull` variant. If the `info.sender` is not the owner, it
     /// returns an `Err` with the `ContractError::Unauthorized` variant. If an admin already exists, it
     /// returns an `Err` with
-    pub fn add_admin(
+    pub fn set_admin(
         &self,
```

Comment:
wont we be using the admin from contract info?

Reply:
admin from contract info is the Owner and has permissions we do not want the admin to have. But removed owner since that will be covered by the internal Admin.